### ISSUE TITLE
Other values not appearing initially in Checkboxes & Dropdown (Multiple) interfaces

### ISSUE THREAD

**Issue by unaussprechlich** [directus/directus#15668]
### Describe the Bug

The Checkboxes/Dropdown (Multiple) interface breaks when custom values are added. 
If the 'Allow Other' is set the false the interface behaves as expected.

![image](https://user-images.githubusercontent.com/9119091/191305780-aac28c4a-a975-419d-b30d-cb13007c402f.png)
Raw value:`[
	"3",
	"Test"
]`

### To Reproduce

Create a Data Model with a Json field and the Checkboxes/Dropdown (Multiple). Enable 'Allow Other' and add some items.

### 1. Before the Item has been saved the Interface looks like this:
![image](https://user-images.githubusercontent.com/9119091/191297308-ec4eaf5b-cdb0-49d7-b270-5057badf9a97.png)
![image](https://user-images.githubusercontent.com/9119091/191304422-2ae9f1ac-b496-44a2-9eb1-27330ad5956d.png)
Raw value: `[
	"Test",
	"3"
]`

### 2. After saving and reloading we get:
![image](https://user-images.githubusercontent.com/9119091/191298122-920aa947-948e-4aa4-8d9b-43e853f4ed59.png)
![image](https://user-images.githubusercontent.com/9119091/191304015-d116144f-5bf0-4493-8c28-dcda8e8f3e21.png)

Raw value: `[
	"Test",
	"3"
]`

### 3. Adding a new value in `+ Other... `
![image](https://user-images.githubusercontent.com/9119091/191298351-3ec59188-256d-4d54-a818-12cba584c79e.png)
Raw value:`[
	"Test2"
]`
![image](https://user-images.githubusercontent.com/9119091/191304597-5b0ac3cf-30c7-42a4-ace1-cfaca1847c7e.png)
Raw value:`[
	"3",
	"Test2"
]`

### What version of Directus are you using?

9.18.0

### What version of Node.js are you using?

16.15.1

### What database are you using?

SQLite

### What browser are you using?

Chome

### How are you deploying Directus?

locally

**Comment by azrikahar** [comment]
Seems like the "Other Values" aren't appearing _on load_, but will re-appear after toggling other options:

### Dropdown (Multiple)

https://user-images.githubusercontent.com/42867097/191417059-2fbb0a78-010b-47ea-acab-e43956b417c9.mp4

### Checkboxes

https://user-images.githubusercontent.com/42867097/191417042-b34d9c17-90a1-453c-a57f-c8ed52051655.mp4

**Comment by unaussprechlich** [comment]
@Nitwel I have checked out https://github.com/directus/directus/pull/15758 locally. It seems like that https://github.com/directus/directus/pull/15758 solved the initial loading Issue by triggering the watcher immediately. 

I discovered another one Issue with the Checkboxes Interface. If you select the text input field of other all previous selected items will be deselected. his is probably caused by emitting "input" in `setup` with the correct values and again on the `v-checkbox` component with the raw values. 

I have created https://github.com/directus/directus/pull/15770 for a potential fix.
